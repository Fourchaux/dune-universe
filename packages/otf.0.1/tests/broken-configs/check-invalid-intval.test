PROG=./otf
ARGS=tests/broken-configs/invalid-intval
EXIT_CODE=1
