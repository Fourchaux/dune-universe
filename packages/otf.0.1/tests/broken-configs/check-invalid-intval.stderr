otf [-print-config|-run-test] <test_name>
Error: Variable "EXIT_CODE" should be an integer
